### PR TITLE
Results: figure 4 (holdout cancer types analysis)

### PR DESCRIPTION
**Pull request by jjc2718**

### CODE REVIEW

File: content/02.main-text.md [hunk 1]
```diff
@@ -137,3 +137,26 @@ This suggests that a _PIK3CA_ mutation status classifier that is more parsimonio
 **E.** _PIK3CA_ mutation status prediction performance generalizing from TCGA (holdout, orange), to CCLE (green), with "best" and "smallest good" models labeled.
 ](images/figure_3.png){#fig:tcga_ccle_smallest_best width="90%"}
 
+### Generalization across cancer types yields similar results to generalization across datasets
+
+To evaluate generalization across biological contexts within a dataset, we trained mutation prediction classifiers on all but one cancer type in TCGA, performed model selection on a holdout set stratified by cancer type, and held out the remaining cancer type as a test set.
+We performed the same "best" vs. "smallest good" analysis that was previously described, across 294 gene/holdout cancer type combinations (Figure {@fig:cancer_type_holdout}A).
+We observed 127/294 gene/cancer type combinations (43.1%) that had better generalization performance with the "best" model, compared to 78/294 (26.5%) for the "smallest good" model, and the other 89 gene/cancer type combinations having the same "best" and "smallest good" model and thus no difference in performance.
```
Comment (cgreene) [inline content/02.main-text.md:144]:
```suggestion
We observed 127/294 gene/cancer type combinations (43.1%) that had better generalization performance with the "best" model, compared to 78/294 (26.5%) for the "smallest good" model.
The other 89 gene/cancer type combinations had the same "best" and "smallest good" model and thus no difference in performance.
```

File: content/02.main-text.md [hunk 2]
```diff
@@ -137,3 +137,26 @@ This suggests that a _PIK3CA_ mutation status classifier that is more parsimonio
...
+This is similar to what we observed for the cross-dataset experiments, with slightly more instances where the "best" model on the stratified holdout data also generalizes the best, but no pronounced distributional shift in either direction.
```
Comment (cgreene) [inline content/02.main-text.md:145]:
```suggestion
This is consistent with our cross-dataset experiments, with slightly more instances where the "best" model on the stratified holdout data also generalizes the best, but no pronounced distributional shift in either direction.
```

**Review by cgreene — APPROVED** [review]
I think this ends up leaving a bit of a cliffhanger. It'll be important to get some interpretation in the conclusions.

**Comment by jjc2718** [comment]
> I think this ends up leaving a bit of a cliffhanger. It'll be important to get some interpretation in the conclusions.

Right, I wasn't sure where exactly to draw the line between results and discussion - I was angling toward having most of the interpretation in the discussion (in progress). We'll have to read through it end-to-end once I have a draft finished, to see if the structure makes sense.

**Pull request merged**